addons/l10n_cl/views/report_invoice.xml

```diff
@@ -208,9 +213,22 @@
             <div class="row">
                 <div name="stamp" class="col-4 text-center"/>
                 <div name="transferable-table" class="col-4"/>
-                <div name="transferable-legend" class="col-4 float-end"/>
+                <div name="transferable-legend" class="col-4 text-end">
+                    <t t-if="o.l10n_latam_document_type_id.code == '39' and o.country_code == 'CL'">
+                        The VAT tax of this boleta is: <span t-esc="o._l10n_cl_get_amounts()['vat_amount']" t-options="{'widget': 'monetary', 'display_currency': o.currency_id}"/>.
+                    </t>
+                </div>
             </div>
         </xpath>
+        <xpath expr="//span[@t-field='line.name']" position="replace">
+            <t t-set="line_amounts" t-value="line._l10n_cl_get_line_amounts()"/>
+            <t t-if="line_amounts.get('second_currency') and not o.l10n_latam_document_type_id._is_doc_type_export()">
+                <span t-esc="'%s (%s: %s @ %s)' % (line.name, o.currency_id.name, o._float_repr_float_round(line.price_unit, 2), o._float_repr_float_round(1/o.currency_id.rate, 2))" t-options="{'widget': 'text'}"/>
```

Comment:
line.price_unit should just need a float_repr

Reply:
we made the same thing for the lines in this commit 0659f4d we call `_l10n_cl_get_line_amounts()`from inside the method `_l10n_cl_prices_and_taxes()`